Changelog.md

```diff
@@ -1,3 +1,15 @@
+## 0.2.6 - 2024-04-12
+
+### Changed
+
+-   Updated Nordic Semiconductor documentation links.
+
+### Fixed
+
+-   A skip button is now shown in the verification step if verification fails.
+-   Modem firmware version v2.0.1 was mistaknly displayed as v2.0.0 for nRF91x1
+    devices.
```

Comment:
```suggestion
-   Modem firmware version v2.0.1 was wrongly displayed as v2.0.0 for nRF9151 and nRF9161
    devices.
```

Reply:
I changed it to say "nRF9151 DK and nRF9161 DK", I hope that's fine.